Lombiq.Tests.UI/Services/UITestExecutionSession.cs

```diff
@@ -140,6 +131,33 @@ public async Task<bool> ExecuteAsync(int retryCount, string dumpRootPath)
             return false;
         }
 
+        private Exception ProcessException(Exception exception)
+        {
+            if (exception is PageChangeAssertionException pageChangeAssertionException)
+            {
+                _testOutputHelper.WriteLineTimestampedAndDebug(pageChangeAssertionException.Message);
+                exception = pageChangeAssertionException.InnerException;
+
+                if (exception is HtmlValidationAssertionException htmlValidationAssertionException &&
+                    htmlValidationAssertionException.HtmlValidationConfiguration.CreateReportOnFailure)
+                {
+                    _testOutputHelper.WriteLineTimestampedAndDebug(
+                        "Check the HTML validation report in the failure dump for details.");
```

Comment:
Why is this not good in the exception, as before? Then we don't need the whole block here.